client/http-client/src/client.rs

```diff
@@ -207,16 +207,19 @@ impl ClientT for HttpClient {
 				}
 			};
```

Comment:
Dumb (unrelated to this PR) question: what's the purpose of this async block?

Reply:
it's part of the `tracing stuff` bug fix, it should be fixed by https://github.com/paritytech/jsonrpsee/pull/846